stable/vpa/templates/admission-controller-certgen.yaml

```diff
@@ -117,5 +117,10 @@ spec:
             # Clean up after we're done.
             echo "Deleting ${TMP_DIR}."
             rm -rf ${TMP_DIR}
+        env:
```

Comment:
If there are no keys, will this still be OK?

Reply:
Yeah, because the default values file specifies `{}`

Reply:
Oh that's what I meant - if it's an empty obect, the yaml ends up looking like
```
foo: bar
env:
baz: quuz
```
Does yaml treat that as `null`?

Reply:
Seems to be the case. Just tested it.